### PR TITLE
Padlock icons in room header

### PR DESCRIPTION
**Pull request by foldleft**
Fixes https://github.com/vector-im/riot-web/issues/11832

Encrypted, locked: Screenshot 2020-01-13 at 14 48 43
Unencrypted, locked: Screenshot 2020-01-13 at 15 04 37
Public room: Screenshot 2020-01-13 at 14 48 55

### CODE REVIEW

File: src/components/views/rooms/RoomHeader.js [hunk 1]
```diff
@@ -159,6 +159,10 @@ module.exports = createReactClass({
             <E2EIcon status={this.props.e2eStatus} /> :
             undefined;
 
+        const joinRules = this.props.room && this.props.room.currentState.getStateEvents("m.room.join_rules", "");
+        const joinRule = joinRules && joinRules.getContent().join_rule;
+        const joinRuleClass = "mx_RoomHeader_PrivateIcon" + (joinRule === "invite" ? " mx_RoomHeader_isPrivate" : "");
```
Comment (t3chguy) [inline src/components/views/rooms/RoomHeader.js]: could this use the classnames module for readibility?

**Comment by jryans** [comment]
@foldleft, since this is a UI change, it would be great to include a screenshot in the PR body. Then we can also get @nadonomy to review it from a design perspective.

**Comment by foldleft** [comment]
> @foldleft, since this is a UI change, it would be great to include a screenshot in the PR body. Then we can also get @nadonomy to review it from a design perspective.

oops, I totally took some

File: src/components/views/rooms/RoomHeader.js (hunk 1 above)
Comment (foldleft) [reply src/components/views/rooms/RoomHeader.js]: oh, neat, I'd not seen that before but it's totally in the rest of hte file

**Comment by jryans** [comment]
> oops, I totally took some

Great, I added @nadonomy as an extra reviewer for the design side.

**Comment by jryans** [comment]
According to our new process experiment discussed yesterday, now that it's guarded by a labs flag, we have the option of skipping design review on the PR. (I'll leave it to @foldleft to decide whether to wait for it or not.)

**Comment by nadonomy** [comment]
Is it guarded by the same labs flag as the general cross-signing one? If so then I don't see a problem with merging. 

When we move cross-signing from /develop to production we'll be doing rounds of small visual polish and there'll be opportunity to action the required polish then.

**Comment by jryans** [comment]
> Is it guarded by the same labs flag as the general cross-signing one? If so then I don't see a problem with merging.

Yes, this and anything cross-signing related are all using a single labs flag.

**Pull request merged**